java/registry/src/main/resources/public/_schemas/Issuer.json

```diff
@@ -0,0 +1,36 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema",
```

Comment:
Schemas added? intentional?

Reply:
it was added by mistake, i have removed it now